v1/timeseries/data_point_insertion_request.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package v1.timeseries;
```

Comment:
`com.cognite.v1.timeseries`

What about following the standard naming convention? We're doing this most other places as far as I know (at least seismic and opint).

Reply:
I added .proto to the package to make it not clash with internal classes in our code so easily